include/FreeRTOSConfig.h

```diff
@@ -181,6 +181,10 @@ void vAssertCalled(unsigned long ulLine, const char * pcFile);
 extern BaseType_t prvRaisePrivilege( void );
 #define RAISE_PRIVILEGE BaseType_t xRunningPrivileged = prvRaisePrivilege ()
 #define RESET_PRIVILEGE if( xRunningPrivileged == 0 ) portSWITCH_TO_USER_MODE()
+
+void initializeProfiler();

```

Comment:
These should be declared in a USER_CODE section instead. They will get overwritten if someone runs halcogen in this code

Reply:
Of course it doesn't show in a a context diff, but the prototypes are in "USER CODE BEGIN (5)"